src/propagation/asteroid_dynamical_models.jl

```diff
@@ -1,11 +1,21 @@
+# Multithreaded version of
+# https://github.com/PerezHz/PlanetaryEphemeris.jl/blob/ed36a2f6e34567a14887cf2811ec7787f124f062/src/interpolation/TaylorInterpolant.jl#L147
+# TO DO: move this to PlanetaryEphemeris.jl
```

Comment:
Despite the fact that this function will be moved, I suggest to add docstrings

Reply:
Done.